tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -0,0 +1,155 @@
+"""
+Module to perform FIO workload
+"""
+import logging
+import pytest
+from elasticsearch import Elasticsearch
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs.ocp import (
+    OCP, get_ocs_version, get_build, get_platform
+)
+from ocs_ci.utility import templating
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest, performance
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def ripsaw(request):
+
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
+    request.addfinalizer(teardown)
+    return ripsaw
+
+
+def analyze_regression(io_pattern, es_username):
+    """
+    Analyzes the FIO result for variance and regression
+    The test fails ff the test run has more than 5% regression
+
+    Args:
+        io_pattern (str): 'sequential' or 'random' workload
+        es_username (str): ocs_build used in the CR object
+
+    """
+    es = Elasticsearch([{'host': constants.ES_SERVER_IP, 'port': constants.ES_SERVER_PORT}])
+    # Todo: Fetch benchmark values for FIO, which
+    #  Will be implemented after finalizing on h/w
+    # fetch results for the current run with unique es_username
+    fio_analyzed_result = es.search(index='ripsaw-fio-analyzed-result',
+                                    body={"query": {"match": {'user': es_username}}})
+    assert fio_analyzed_result['hits']['hits'], 'Results not found in Elasticsearch'
+    for result in fio_analyzed_result['hits']['hits']:
+        test_data = result['_source']['ceph_benchmark_test']['test_data']
+        object_size = test_data['object_size']
+        operation = test_data['operation']
+        total_iops = test_data['total-iops']
+        log.info(
+            f"io_pattern: {io_pattern}\n"
+            f"block_size: {object_size}\n"
+            f"operation: {operation}\n"
+            f"total_iops: {total_iops}\n"
+        )
+        # Fail test if std deviation is above 5%
+        # Todo: Remove the below skip for random workload once
+        #  https://github.com/cloud-bulldozer/snafu/issues/180 is fixed
+        if io_pattern == 'sequential':
+            std_dev = 'std-dev-' + object_size
+            variance = test_data[std_dev]
+            log.warning(f'variance - {variance} is greater than 5%')
+        # Todo: Fail test if 5% deviation from benchmark value
+
+
+@pytest.mark.skipif(
```

Comment:
@keesturam can you use this mark instead https://github.com/red-hat-storage/ocs-ci/pull/2119 , I will update the final comment I have there. The travis is failing due to get_plaftorm call